auditbeat/module/file_integrity/_meta/config.yml.tmpl

```diff
@@ -44,6 +44,16 @@
   - '/\.git($|/)'
   {{- end }}
 
+  # List of regular expressions used to explicitly include files. When configured,
+  # Auditbeat will ignore files unless they match a pattern.
+  {{ if eq .GOOS "windows" -}}
+  #include_files:
+  #- '\\\.ssh($|\)'
```

Comment:
I *think* the last \ needs to be escaped too:

'\\\.ssh($|\\)'

I think I tested it right, but with the many levels of escaping required from yaml I might be wrong
https://play.golang.org/p/6EcInvdHT-g

Reply:
Think you're right - it definitely fails without it on Mac. I can sort of "test it" by creating a directory named `\.ssh\` and that works as expected.